components/commands/browser/BUILD.gn

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2023 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+component("browser") {
+  output_name = "commands_browser"
+  defines = [ "IS_COMMANDS_BROWSER_IMPL" ]
+  sources = []
```

Comment:
looks like this doesn't have any sources, which makes this component fail to build on windows. You can either remove this or add some `.cc` file that will make it compile into something usable.

Reply:
Thanks @goodov - it would've taken me ages to work that out